src/main/resources/ath-container/Dockerfile

```
@@ -55,6 +58,23 @@ RUN curl -ffSLO https://dlcdn.apache.org/maven/maven-3/$MAVEN_VERSION/binaries/a
     mv /opt/apache-maven-* /opt/maven
 ENV PATH="$PATH:/opt/maven/bin"
 
+# Docker installation according to https://docs.docker.com/engine/install/ubuntu/
+RUN mkdir -m 0755 -p /etc/apt/keyrings
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+RUN echo \
+      "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+      $(lsb_release -cs) stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null
```

```suggestion
      $(lsb_release -cs) stable" > /etc/apt/sources.list.d/docker.list
```

the original doc has `| sudo tee > dev/null` because the useris not running as root / has no permissions on `/etc/apt/sources.list`  so running tee as root means it ca write to the file and std.out (and std.out is thrown away) - we can just redirect std.out to the file.